README.md

```diff
@@ -63,31 +63,32 @@ The site should open in a new page in your browser at the address: [localhost:30
 
 The Data Dictionary content is not updated dynamically when the dictionary is updated. Instead, the dictionary content is included as static files included in this repository. This allows the docs site to be completely static and not have to load the dictionary from the source everytime it is requested.
 
-To simplify updating the dictonary content, the process has been scripted so it can be run by following the commands below.
+To simplify updating the dictonary content, the process has been scripted so it can be run by following the commands below. Files will be saved in `scripts/data` folder and then moved to website directory by `copy-data` script.
```

Comment:
just for my info, why do we check in scripts/data to git ?

Reply:
Only reason I have is that the script uses `schema-versions.json` in `scripts/data` to run, so it also uses whatever is already there to save diffs.

It does mean that the same files are in `scripts` and `website` but I like having them loosely coupled in the hopes we can generalise our `website` some more to be a boilerplate for docs in the future.